shared/src/business/entities/JoiValidationDecorator.ts

```diff
@@ -120,12 +120,12 @@ function getFormattedValidationErrors(entity) {
   return Object.keys(obj).length === 0 ? null : obj;
 }
 
-export interface TValidationEntity {
+export interface TValidationEntity<T> {
```

Comment:
Love the generic.

Not sure what you want to do in the project but generally I have seen interface names prepended with an "I". So if I had a food interface it would be IFood. Wanted to see if you wanted to establish a similar pattern here. so have IValidationEntity<T>.